Apps/AdminWebClient/src/ClientApp/src/views/Support.vue

```diff
@@ -91,33 +74,29 @@ export default class SupportView extends Vue {
         return Object.keys(QueryType).filter((x) => isNaN(Number(x)) !== false);
     }
 
+    private getEmailOrSms(email: Email | null, sms: string | null): string {
+        let emailTo = email !== null ? email.to : "N/A";
+        let smsValue = sms !== null ? sms : "N/A";
+        return emailTo && smsValue === "N/A" ? "N/A" : `${emailTo}/${smsValue}`;
```

Comment:
There won't be both an email and SMS at the same time, so the value in the column should be whichever one is populated (instead of both displayed with a slash).  Will confirm.

Reply:
I think that's correct - each row is an entry for a particular sms or email verification, so you don't need N/A.